web-marketplace/src/pages/Projects/AllProjects/AllProjects.SideFilter.tsx

```diff
@@ -57,6 +61,16 @@ export const SideFilter = ({
   const [creditClassSelectedFilters, setCreditClassSelectedFilters] = useAtom(
     creditClassSelectedFiltersAtom,
   );
+
+  useEffect(() => {
```

Comment:
instead of resetting the `creditClassSelectedFilters` in a different `useEffect` you can just do that in the existing one in https://github.com/regen-network/regen-web/blob/dev/web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx#L114, checking whether `creditClassFilter.path` is in your list of credit classes to deselect or not
that will avoid the behavior that @clevinson is describing here: https://github.com/regen-network/regen-web/pull/2365#issuecomment-2111536879

Reply:
Thanks for the tip, I just refactored based on your suggestion and it's a more concise solution.